org.faktorips.build.config/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.faktorips</groupId>
+	<artifactId>org.faktorips.build.config</artifactId>
+	<version>1.2.5</version>
```

Comment:
Hier bin ich nicht ganz sicher... wenn dieses Modul Teil des größeren Projekts ist finde ich eine unabhängige Versionierung nicht so schön. Sollte hier die gleiche SNAPSHOT-Version wie in den anderen Modulten stehen und dieses Modul auch im aggregator/pom.xml bzw. dem neuen base pom.xml aus #13 enthalten sein?

Reply:
Ja die version würde ich dann aus dem Base-pom übernehmen, hier soll es erstmal darum gehen den code überhaupt ins repository zu kriegen sodass der Build das nutzen kann.

Reply:
Ich könnte auch die beiden PRs mergen wenn du denkst das das besser ist?

Reply:
Ne, dann machen wir das gerne in zwei Schritten.